pkg/tsdb/azuremonitor/loganalytics/azure-log-analytics-datasource.go

```diff
@@ -680,7 +705,7 @@ func encodeQuery(rawQuery string) (string, error) {
 	return base64.StdEncoding.EncodeToString(b.Bytes()), nil
 }
 
-func buildTracesQuery(operationId string, traceTypes []string, filters []types.TracesFilters, resultFormat *string, resources []string) string {
+func buildTracesQuery(operationId string, parentSpanID *string, traceTypes []string, filters []types.TracesFilters, resultFormat *string, resources []string) string {
```

Comment:
The placement of the arg is interesting here. Why not just append it to the end of the list of args?

Reply:
I feel like it just felt natural having it next to the `operationId` but I'm not picky 😄

Reply:
That was my guess! :raised_hands: Fine by me, just curious.